src/acceptance/app/app_suite_test.go

```diff
@@ -141,3 +142,14 @@ func averageMemoryUsedByInstance(appGUID string, timeout time.Duration) uint64 {
 
 	return memSum / uint64(len(memoryUsedArray))
 }
+
+func appReport(appName string, timeout time.Duration) {
+	Eventually(cf.Cf("app", appName, "--guid"), timeout).Should(Exit())
+	Eventually(cf.Cf("logs", appName, "--recent"), timeout).Should(Exit())
+}
+
+func readPolicyFromFile(filename string) []byte {
```

Comment:
Personally this shouldn't be a method unless we are really out to save 1 line of code everywhere.